kernel/src/common/hardwareinfo.c

```diff
@@ -72,3 +77,8 @@ void print_hardwareinfo() {
 
     kprintf("==============================================\n");
 }
+
+bool address_in_reserved_region(size_t address) {
```

Comment:
Usually, in c when you're doing 'object oriented' like programming (something this kernel does all the time), you give it the 'self' object as first parameter. I know hardwareinfo is a singleton which you can just get, but it may be nice to still give it the hardwareinfo and determine it for that info struct. This is your, and the others of this year's group's decision

(so what do others think?)

Reply:
Personally, I would prefer the way that Francisco wrote it since it is more similar to the traditional and dogmatic way of writing C. However, if this is the prevailing style used throughout the code base, we would be better off following that in our year too. The important thing in code standards isn't the one you use after all, it is being consistent.